qwandaq/src/main/java/life/genny/qwandaq/models/GennyToken.java

```diff
@@ -119,10 +120,10 @@ public void init(String token) {
 		String azp = getString("azp");
 
 		// If azp is alyson then it must be internmatch
-		// NOTE: solution would be to eliminate any internmatch 
+		// NOTE: solution would be to eliminate any internmatch
 		// tokens using this client id.
-		if ("alyson".equals(azp)) {
-			//log.warn("alyson clientid detected.");
+		if (("alyson".equals(azp)) || ("backend".equals(azp))) {
```

Comment:
This check could be a part of some singleton util class. It would be easier to use the util throughout the code, and change at a single place if required.